src/IECoreImage/ClientDisplayDriver.cpp

```diff
@@ -96,23 +96,23 @@ ClientDisplayDriver::ClientDisplayDriver( const Imath::Box2i &displayWindow, con
 	m_data->m_host = displayHostData->readable();
 	m_data->m_port = displayPortData->readable();
 
-	tcp::resolver resolver(m_data->m_service);
-	tcp::resolver::query query(m_data->m_host, m_data->m_port);
-
-	boost::system::error_code error;
-	tcp::resolver::iterator iterator = resolver.resolve( query, error );
-	if( !error )
-	{
+        boost::asio::io_context io_context;
```

Comment:
Why make a new `io_context` here, rather than using `m_data->m_service` as before? Since the old `asio::io_service` was just a typedef for `asio::io_context`, I wouldn't expect anything to need to change here.